language/German/strings.po

```diff
@@ -4565,1083 +4626,1083 @@ msgstr ""
 msgctxt "input-name"
 msgid "Mouse X 5"
-msgstr ""
+msgstr "Maus X 5"
 
 #: src/emu/inpttype.ipp:845
 msgctxt "input-name"
 msgid "Mouse X 6"
-msgstr ""
+msgstr "Maus X 6"
 
 #: src/emu/inpttype.ipp:846
 msgctxt "input-name"
 msgid "Mouse X 7"
-msgstr ""
+msgstr "Maus X 7"
 
 #: src/emu/inpttype.ipp:847
 msgctxt "input-name"
 msgid "Mouse X 8"
-msgstr ""
+msgstr "Mouse X 8"
 
 #: src/emu/inpttype.ipp:848
 msgctxt "input-name"
 msgid "Mouse X 9"
-msgstr ""
+msgstr "Maus X 9"
 
 #: src/emu/inpttype.ipp:849
 msgctxt "input-name"
 msgid "Mouse X 10"
-msgstr ""
+msgstr "Maus X 10"
 
 #: src/emu/inpttype.ipp:854
 msgctxt "input-name"
 msgid "Mouse Y"
-msgstr ""
+msgstr "Maus Y"
 
 #: src/emu/inpttype.ipp:855
 msgctxt "input-name"
 msgid "Mouse Y 2"
-msgstr ""
+msgstr "Maus Y 2"
 
 #: src/emu/inpttype.ipp:856
 msgctxt "input-name"
 msgid "Mouse Y 3"
-msgstr ""
+msgstr "Maus Y 3"
 
 #: src/emu/inpttype.ipp:857
 msgctxt "input-name"
 msgid "Mouse Y 4"
-msgstr ""
+msgstr "Maus Y 4"
 
 #: src/emu/inpttype.ipp:858
 msgctxt "input-name"
 msgid "Mouse Y 5"
-msgstr ""
+msgstr "Maus Y 5"
 
 #: src/emu/inpttype.ipp:859
 msgctxt "input-name"
 msgid "Mouse Y 6"
-msgstr ""
+msgstr "Maus Y 6"
 
 #: src/emu/inpttype.ipp:860
 msgctxt "input-name"
 msgid "Mouse Y 7"
-msgstr ""
+msgstr "Maus Y 7"
 
 #: src/emu/inpttype.ipp:861
 msgctxt "input-name"
 msgid "Mouse Y 8"
-msgstr ""
+msgstr "Maus Y 8"
 
 #: src/emu/inpttype.ipp:862
 msgctxt "input-name"
 msgid "Mouse Y 9"
-msgstr ""
+msgstr "Maus Y 9"
 
 #: src/emu/inpttype.ipp:863
 msgctxt "input-name"
 msgid "Mouse Y 10"
-msgstr ""
+msgstr "Maus Y 10"
 
 #: src/emu/inpttype.ipp:868
 msgctxt "input-name"
 msgid "Keypad"
-msgstr ""
+msgstr "Ziffernblock"
 
 #: src/emu/inpttype.ipp:869
 msgctxt "input-name"
 msgid "Keyboard"
-msgstr ""
+msgstr "Tastatur"
 
 #: src/emu/inpttype.ipp:874
 msgctxt "input-name"
 msgid "On Screen Display"
-msgstr ""
+msgstr "Bildschirmanzeige"
 
 #: src/emu/inpttype.ipp:875
 msgctxt "input-name"
 msgid "Break in Debugger"
-msgstr ""
+msgstr "In Debugger verzweigen"
 
 #: src/emu/inpttype.ipp:876
 msgctxt "input-name"
 msgid "Config Menu"
-msgstr ""
+msgstr "Konfigurationsmenü"
 
 #: src/emu/inpttype.ipp:877
 msgctxt "input-name"
 msgid "Pause"
-msgstr ""
+msgstr "Pause"
 
 #: src/emu/inpttype.ipp:878
 msgctxt "input-name"
 msgid "Pause - Single Step"
-msgstr ""
+msgstr "Pause - Einzelschritt"
 
 #: src/emu/inpttype.ipp:879
 msgctxt "input-name"
 msgid "Rewind - Single Step"
-msgstr ""
+msgstr "Zurückspulen - Einzelschritt"
 
 #: src/emu/inpttype.ipp:880
 msgctxt "input-name"
 msgid "Reset Machine"
-msgstr ""
+msgstr "Maschinenreset"
 
 #: src/emu/inpttype.ipp:881
 msgctxt "input-name"
 msgid "Soft Reset"
-msgstr ""
+msgstr "Softreset"
 
 #: src/emu/inpttype.ipp:882
 msgctxt "input-name"
 msgid "Show Decoded Graphics"
-msgstr ""
+msgstr "Decodierte Grafiken anzeigen"
 
 #: src/emu/inpttype.ipp:883
 msgctxt "input-name"
 msgid "Frameskip Dec"
-msgstr ""
+msgstr "Frames überspringen dekrement"
```

Comment:
argh. I thought 1 solid minute about that but I couldn't find a better way. I still don't like the "inkrement"/"dekrement" part, maybe "erhöhen/verringern" would be better here. But the "Frames überspringen" part is annoying as I couldn't find a better tranlation, yet it sounds so contrieved....

Reply:
I've changed it to erhöhen/verringern in order to be consistent with the other changes.